wenet/transformer/asr_model.py

```diff
@@ -575,20 +640,36 @@ def ctc_activation(self, xs: torch.Tensor) -> torch.Tensor:
         """
         return self.ctc.log_softmax(xs)
 
+    @torch.jit.export
+    def is_bidirectional_decoder(self) -> bool:
+        """
+        Returns:
+            torch.Tensor: decoder output
+        """
+        if hasattr(self.decoder, 'right_decoder'):
+            return True
+        else:
+            return False
+
     @torch.jit.export
     def forward_attention_decoder(
```

Comment:
@torch.jit.export
add forward_attention_reverse_decoder for interface compatibility